internal/p2p/metrics.go

```diff
@@ -27,15 +27,32 @@ var (
 
 // Metrics contains metrics exposed by this package.
 type Metrics struct {
-	// Number of peers.
-	Peers metrics.Gauge
+	// Number of peers connected.
+	PeersConnected metrics.Gauge
```

Comment:
While I think that this is a better name, I think we shouldn't do it in a point release. Dashboards for v0.35 already exist with the `peers` name and I don't think we should break them.

Reply:
I've made this change, but I think the name as exported to dashboards was still peers because the name is specified statically..

Reply:
oh yeah :facepalm: